.github/workflows/buf-release.yml

```
@@ -0,0 +1,28 @@
+      # Push the protobuf definitions to the BSR
+      - uses: bufbuild/buf-push-action@v1
+        with:
+          buf_token: ${{ secrets.BUF_TOKEN }}
+      - name: "push the tag label to BSR"
+        run: |
+          set -euo pipefail
+
+          bash ./scripts/buf-install.sh
+
+          echo ${{ secrets.BUF_TOKEN }} | buf registry login --token-stdin
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)